### DISCUSSION TITLE
Gas tank water heater with Econet

### DISCUSSION THREAD

**Discussion by dualtronic** [esphome-econet/esphome-econet#374] [General]
Does this solution work for a standard Rheem gas tank water heater with the econet device connected?  It looks the same as the econet device for the electric take water heater, but is specific for the tank gas water heater.

**Comment by thezfunk** [comment]
I am wondering the same thing.  The Econet WiFi module stoped connecting to my network and I was sent here as an alternative but I don't see the gas water heater as a supported unit and my connector is different.  I would imagine the wires are the same though?
